docs/alternator/compatibility.md

```diff
@@ -183,7 +183,7 @@ feature's implementation is still subject to change and upgrades may not be
 possible if such a feature is used. For these reasons, experimental features
 are not recommended for mission-critical uses, and they need to be
 individually enabled with the "--experimental-features" configuration option.
-See [Enabling Experimental Features](#yaml_enabling_experimental_features) for details.
+See [Enabling Experimental Features](https://docs.scylladb.com/stable/operating-scylla/admin.html#enabling-experimental-features) for details.
```

Comment:
I've updated the link with the relative path and the section header, following the MD syntax. 
I'd rather not link to the entire file.
Anyway, it will be replaced once we have a dedicated page, and then linking to that page will be easy.

Reply:
Can't you add a anchor (`#something`) to a the end of a relative link instead of an absolute link?
My point was about making the link relative - not about using or not using an anchor.

Reply:
My impression is that you've added this comment to an outdated file... How is this possible?
The link is now:
See [Enabling Experimental Features](/operating-scylla/admin#enabling-experimental-features) for details.

It's working, and it's relative. Is that what you meant?

Reply:
Yes, this is what happens when you add fixes as a separate patch at the end :-)
Thanks, I see now it's relative. Yes, thanks, that's what I meant. Or almost: I am not sure about the "/" in the beginning, but if it works (both on the website and on github) then it works.

Reply:
"/" is necessary for the link to work.
BUT, I can see it's not working on GitHub now... :( It looks like there's no good way to link to a subsection if there's a cross-link between an MD and and RST files, and we want to make it work both on the page (RST-dependent) and GitHub (MD-dependent).

Reply:
> Yes, this is what happens when you add fixes as a separate patch at the end :-)

Can you explain to me how I should add fixes? What's a good practice? Previously, I added several commits and then squashed them.

Reply:
Yes, squashing is best, usually. To just one patch or to several when it makes sense.

Reply:
According to https://gist.github.com/TobiasPalludan/5948533e97d260a04ea47e44ea7cc6b0, a link like `github-tips.md#anchors` to an anchor in some other file should work. I guess I'm missing some details on why it doesn't work, I'll let you figure it out.

I guess it's fine that some of the more intricate tricks don't work on github and only work properly on our site. We should just try to minimize the amount of these problems, I guess.